javascript/javascript2/week3/task3.js

```diff
@@ -0,0 +1,88 @@
+
+
+
+const refs = {
+    input: document.querySelector('[data-input=value]'),
+    button: document.querySelector('.btn-start'),
+    outputS: document.querySelector('.outputS'),
+    outputL: document.querySelector('.outputL'),
+    timeOver: document.querySelector('.timer-over'),
+    divS: document.querySelector('.gamer-s'),
+    divL: document.querySelector('.gamer-l'),
+    
+}
+let inputValue = 0;
+let counterS = 0;
+let counterL = 0;
+
+refs.button.addEventListener('click', onClickBtnStart);
+document.addEventListener('keyup', onKeyupCounter)
+
+
+
+function onClickBtnStart(event) {
+    inputValue = refs.input.value;
+    if (inputValue === "") {
+       return refs.timeOver.textContent = 'Set timeout'
+    }
+    const timeoutId = setTimeout( defineWinner,
+        inputValue * 1000)
+        
+        if (inputValue !== "") {
+         return refs.timeOver.textContent = ''
+     }
+}
+
+function defineWinner() {
+       if (counterL === counterS) {
+           alert('A draw')
+       }  
+       if (counterS > counterL) {
+           renderCanvasEl(refs.divS);
+            setConfetti() 
+       }
+       if (counterL > counterS) {
+           renderCanvasEl(refs.divL);
+           setConfetti() 
+       }
+
+    refs.timeOver.textContent = 'Game over';
+    document.removeEventListener('keyup', onKeyupCounter);
+}
+
+
+function onKeyupCounter(event) {
+   
+    if (!event.code === 'KeyS' & !event.code === 'KeyL') {
```

Comment:
Interesting use of `event.code`
In this case it might be better to use `event.key` as we specifically instruct people so press S and L keys. 
If someone has a different keyboard layout where S and L keys are in some other place (and I'm not even sure such keybords exists :) it might work weird/different. You can see the demo here: https://codepen.io/denilsonsa/pen/epmoma